algebird-core/src/main/scala/com/twitter/algebird/HyperLogLog.scala

```diff
@@ -338,6 +410,17 @@ case class DenseHLL(bits: Int, v: IndexedSeq[Byte]) extends HLL {
       idx += 1
     }
   }
+
+  override def downsize(reducedBits: Int): HLL = {
+    require(reducedBits > 3 && reducedBits < bits, s"Use at least 4, but less than $bits bits")
+    val reducedSize = 1 << reducedBits
+    val reducedV = Array.fill[Byte](reducedSize)(0: Byte)
+    for (i <- 0 until reducedSize) {
+      val newRhoW = getNewRhoWAt(i, reducedBits, v(_))
+      reducedV.update(i, newRhoW.toByte)
+    }
+    DenseHLL(reducedBits, reducedV.toIndexedSeq)
```

Comment:
can you use IndexedSeqArrayByte here to avoid the copy that toIndexedSeq would do?